easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-4.0-foss-2016b.eb

```diff
@@ -0,0 +1,38 @@
+name = 'OpenFOAM'
+version = '4.0'
```

Comment:
@iotaka I just noticed, you mentioned doing a manual install of version 4.1 because this easyconfig isn't working on your system due to problem with Qt4 vs Qt5

Could it be that the issue is fixed because you were installing OF v4.1 by hand, but v4.0 with this easyconfig?

Maybe the build issue is fixed in v4.1?